src/handlers/localstorage/nonceManager.ts

```diff
@@ -0,0 +1,34 @@
+import { getGlobal, saveGlobal } from './common';
+
+export const NONCE_MANAGER = 'nonceManager';
+
+interface NetworkNonceInfo {
+  nonce: number;
+}
+interface AccountNonceInfo {
+  [key: string]: NetworkNonceInfo;
+}
+
+export interface NonceManager {
+  [key: string]: AccountNonceInfo;
+}
+
+export interface NonceManagerUpdate {
+  network: string;
+  account: string;
+  nonce: number;
+}
+
+// -- Constants --------------------------------------- //
+export const NONCE_MANAGER_LOAD_SUCCESS = 'NONCE_MANAGER_LOAD_SUCCESS';
```

Comment:
any reason these consts don't just stay in the redux file?

Reply:
Nope. Forgot to delete when moving types around.